src/components/CreateMenu.js

```diff
@@ -85,26 +85,29 @@ class CreateMenu extends PureComponent {
                         ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED
                         : CONST.MODAL.MODAL_TYPE.POPOVER
                 }
+                anchorPosition={styles.createMenuPosition}
             >
-                {menuItemData.map(({icon, text, onPress}) => (
-                    <Pressable
-                        key={text}
-                        onPress={onPress}
-                        style={({hovered}) => ([
-                            styles.createMenuItem,
-                            hovered && {backgroundColor: themeColors.buttonHoveredBG},
-                        ])}
-                    >
-                        <View style={styles.createMenuIcon}>
-                            <Icon src={icon} />
-                        </View>
-                        <View style={styles.justifyContentCenter}>
-                            <Text style={[styles.createMenuText, styles.ml3]}>
-                                {text}
-                            </Text>
-                        </View>
-                    </Pressable>
-                ))}
+                <View style={this.props.isSmallScreenWidth ? {} : styles.createMenuContainer}>
```

Comment:
The styles for the CreateMenu popover are only relevant on larger screen widths, because on smaller screens it is not a popover.

Reply:
Coming back to this. I think this works for a generic "popover" interface, but it would be cool to achieve this by passing a new modal type rather than have to think about the "container" and "anchorPosition" in the code here.

Reply:
I think that the "container" and "popoverAnchorPosition" specified here are really just styles. They're very specific to the content of _this_ popover.

Reply:
Gonna resolve this since it seems like we are not doing it.